player.js

```diff
@@ -15,7 +15,7 @@ class Player {
         this.color = color;
     }
 
-    draw(context) {
+    drawPlayer(context) {
```

Comment:
A player and meteorite are both "drawn" per say. If I use the same naming convention in the Meteorite class and in Player then there will be an error I believe. A way to prevent the error is to differentiate the two.

Reply:
Updated.